### DISCUSSION TITLE
F11 and F12 now ganged together?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#6054] [Q&A] [answered]
A couple of months or so ago, I upgraded from rather old version of Xournalpp (1.2.0 I think) to 1.2.3 and I discovered that the F11 "full screen" key map also toggles the F12 "show sidebar" functionality. I assumed that this was a bug. I can't find such a reported bug in the Issues DB though.

I took a look at the 1.2.4 release this morning and I see that it has the same behaviour. 

So I'm wondering if this is not a bug but a deliberate design change. If that is the case, could someone please explain the rationale? I find it to be very annoying, but perhaps there is some good reason for this behaviour that I'm not aware of.

**Comment by rolandlo** [comment, marked as answer]
This is a preference that the user can choose. See View tab in menu `Edit > Preferences`.

**Reply by NelkonParker** [reply]
OK. Thanks. That clears things up.
